components/timesheets/TimesheetTable.tsx

```diff
@@ -0,0 +1,232 @@
+        if (id) {
+            const employee = employees?.find((el) => el.id === Number(id));
+            setState({
+                ...state,
+                user: { ...employee },
+            });
+        }
+    };
+
+    return (
+        <Flex
+            flexDirection="column"
+            backgroundColor="white"
+            border="1px solid"
+            borderColor="gray.400"
+            borderRadius="0.2rem"
+            padding="1rem 1rem"
+            marginTop="1.5rem"
+        >
+            {employeesLoading && <Loading></Loading>}
+            {employeeError && (
+                <ErrorAlert
+                    title="Error loading data"
+                    message="Could not load the required data from the server"
+                ></ErrorAlert>
+            )}
+            <Heading as="h2" size="md">
+                Users
+            </Heading>
+            {timesheets && timesheets?.length == 0 && (
+                <Box borderWidth="1px" padding="1rem" margin="1rem">
+                    No users in this project.
+                    <br />
+                    To add a user click the button below.
+                </Box>
+            )}
+            {timesheets && timesheets?.length !== 0 && (
+                <Box borderWidth="1px" padding="1rem" margin="1rem">
+                    <Table variant="simple">
+                        <Thead>
+                            <Tr>
+                                <Th>Name</Th>
+                                <Th>Allocation</Th>
+                                <Th></Th>
+                            </Tr>
+                        </Thead>
+                        <Tbody>
+                            {timesheets.map((el, idx) => {
+                                return (
+                                    <Tr _hover={{ backgroundColor: 'gray.200', cursor: 'pointer' }} key={idx}>
+                                        <Td>
+                                            {el.employee?.first_name} {el.employee?.last_name}
+                                        </Td>
+                                        <Td>{el.allocation} %</Td>
+                                        <Td>
+                                            <Button>x</Button>
+                                        </Td>
+                                    </Tr>
+                                );
+                            })}
+                        </Tbody>
+                    </Table>
+                </Box>
+            )}
+            <Flex flexDirection="row-reverse">
+                <Button colorScheme="blue" onClick={onOpen}>
+                    Add User
+                </Button>
+            </Flex>
+            <Modal isOpen={isOpen} onClose={onClose}>
+                <ModalOverlay />
+                <ModalContent>
+                    <ModalHeader>Add user to project</ModalHeader>
+                    <ModalCloseButton />
+                    <FormControl>
```

Comment:
Please add left and right padding to this form. 0.5 rem should be fine on both sides.

Reply:
![Screenshot from 2021-12-08 10-44-52](https://user-images.githubusercontent.com/17109116/145177305-3fc89e18-6d7c-41af-9572-628c0988e554.png)
added in 727f11d